1501639590357	browserwindow.syncui	DEBUG	observed: weave:ui:sync:finish
1501639591279	browserwindow.syncui	DEBUG	loginFailed has sync state=success.login
1501639591282	browserwindow.syncui	DEBUG	loginFailed has sync state=success.login
1501640190360	Sync.Service	DEBUG	User-Agent: Firefox/54.0.1 (Linux x86_64) FxSync/1.56.0.20170630112252.desktop
1501640190361	Sync.Service	INFO	Starting sync at 2017-08-01 22:16:30 in browser session dpri-lQGxGck
1501640190363	browserwindow.syncui	DEBUG	observed: weave:service:sync:start
1501640190365	browserwindow.syncui	DEBUG	onActivityStart
1501640190369	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1501640190371	Sync.Telemetry	TRACE	observed weave:service:sync:start 
1501640190372	Sync.Status	INFO	Resetting Status.
1501640190372	Sync.Status	DEBUG	Status.service: success.status_ok => success.status_ok
1501640190375	Sync.BrowserIDManager	DEBUG	_ensureValidToken already has one
1501640190384	browserwindow.syncui	DEBUG	loginFailed has sync state=success.login
1501640190402	browserwindow.syncui	DEBUG	loginFailed has sync state=success.login
1501640190442	Sync.Resource	WARN	GET request to https://sync-487-us-west-2.sync.services.mozilla.com/1.5/72634235/info/collections failed: [Exception... "NS_ERROR_UNKNOWN_HOST"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "JS frame :: resource://services-sync/resource.js :: Channel_onStopRequest :: line 509"  data: no] Stack trace: Channel_onStopRequest()@resource://services-sync/resource.js:509 < waitForSyncCallback()@resource://services-common/async.js:98 < Res__request()@resource://services-sync/resource.js:386 < Res_get()@resource://services-sync/resource.js:415 < _fetchInfo()@resource://services-sync/service.js:500 < sync()@resource://services-sync/stages/enginesync.js:82 < onNotify()@resource://services-sync/service.js:1115 < WrappedNotify()@resource://services-sync/util.js:164 < WrappedLock()@resource://services-sync/util.js:120 < _lockedSync()@resource://services-sync/service.js:1105 < sync/<()@resource://services-sync/service.js:1097 < WrappedCatch()@resource://services-sync/util.js:95 < sync()@resource://services-sync/service.js:1086
1501640190448	Sync.Status	DEBUG	Status.sync: success.sync => error.login.reason.network
1501640190448	Sync.Status	DEBUG	Status.service: success.status_ok => error.sync.failed
1501640190449	browserwindow.syncui	DEBUG	observed: weave:service:sync:error
1501640190449	browserwindow.syncui	DEBUG	onActivityStop
1501640190449	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms.
1501640190450	Sync.ErrorHandler	ERROR	Sync encountered an error: Error: NS_ERROR_UNKNOWN_HOST (resource://services-sync/resource.js:395:19) JS Stack trace: waitForSyncCallback@async.js:98:7 < Res__request@resource.js:386:14 < Res_get@resource.js:415:12 < _fetchInfo@service.js:500:14 < sync@enginesync.js:82:16 < onNotify@service.js:1115:7 < WrappedNotify@util.js:164:21 < WrappedLock@util.js:120:16 < _lockedSync@service.js:1105:12 < sync/<@service.js:1097:7 < WrappedCatch@util.js:95:16 < sync@service.js:1086:5 < Res_get@resource.js:415:12 < _fetchInfo@service.js:500:14 < sync@enginesync.js:82:16 < onNotify@service.js:1115:7 < WrappedNotify@util.js:164:21 < WrappedLock@util.js:120:16 < _lockedSync@service.js:1105:12 < sync/<@service.js:1097:7 < WrappedCatch@util.js:95:16 < sync@service.js:1086:5
1501640190453	Sync.Telemetry	TRACE	observed weave:service:sync:error 
1501640190458	Sync.Service	DEBUG	Exception calling anonymous function: Error: NS_ERROR_UNKNOWN_HOST (resource://services-sync/resource.js:395:19) JS Stack trace: waitForSyncCallback@async.js:98:7 < Res__request@resource.js:386:14 < Res_get@resource.js:415:12 < _fetchInfo@service.js:500:14 < sync@enginesync.js:82:16 < onNotify@service.js:1115:7 < WrappedNotify@util.js:164:21 < WrappedLock@util.js:120:16 < _lockedSync@service.js:1105:12 < sync/<@service.js:1097:7 < WrappedCatch@util.js:95:16 < sync@service.js:1086:5 < Res_get@resource.js:415:12 < _fetchInfo@service.js:500:14 < sync@enginesync.js:82:16 < onNotify@service.js:1115:7 < WrappedNotify@util.js:164:21 < WrappedLock@util.js:120:16 < _lockedSync@service.js:1105:12 < sync/<@service.js:1097:7 < WrappedCatch@util.js:95:16 < sync@service.js:1086:5
1501640190470	Sync.ErrorHandler	DEBUG	Addons installed: 9
1501640190470	Sync.ErrorHandler	DEBUG	 - Dark Theme for YouTubeâ„¢, version 0.1.6, id jid1-y4pUQrD6t1PJJa@jetpack
1501640190471	Sync.ErrorHandler	DEBUG	 - YouTube Plus, version 1.8.7, id particle@particlecore.github.io
1501640190471	Sync.ErrorHandler	DEBUG	 - Tab Groups, version 2.1.4, id tabgroups@quicksaver
1501640190471	Sync.ErrorHandler	DEBUG	 - Vimperator, version 3.16.0, id vimperator@mozdev.org
1501640190471	Sync.ErrorHandler	DEBUG	 - InvisibleHand, version 3.9.91, id canitbecheaper@trafficbroker.co.uk
1501640190472	Sync.ErrorHandler	DEBUG	 - Easy Youtube Video Downloader Express, version 9.16, id {b9acf540-acba-11e1-8ccb-001fd0e08bd4}
1501640190472	Sync.ErrorHandler	DEBUG	 - TableTools2, version 1.17.1-signed.1-signed, id tabletools2@mingyi.org
1501640190472	Sync.ErrorHandler	DEBUG	 - NoSquint Plus, version 54.0, id zoomlevelplus@zoomlevelplus.net
1501640190473	Sync.ErrorHandler	DEBUG	 - HTTPS Everywhere, version 5.2.21, id https-everywhere@eff.org
